src/go/k8s/kind.yaml

```diff
@@ -2,20 +2,20 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
-  image: vectorizedcloud/kindestnode:v1.21.1-pr2515
+  image: kindest/node:v1.24.0
```

Comment:
There is also [v1.24.7](https://hub.docker.com/layers/kindest/node/v1.24.7/images/sha256-5c015142d9b60a0f6c45573f809957076514e38ec973565e2b2fe828b91597f5?context=explore) available, it might be worth using that latest patch version.

Reply:
OFC :)